exchange-common/src/main/scala/com/vesoft/exchange/common/config/SourceConfigs.scala

```diff
@@ -157,6 +158,37 @@ case class MySQLSourceConfigEntry(override val category: SourceCategory.Value,
   }
 }
 
+/**
+ * PostgreSQLSourceConfigEntry
+ *
+ * @param category
+ * @param host
+ * @param port
+ * @param database
+ * @param table
+ * @param user
+ * @param password
+ * @param sentence
+ */
+case class PostgresSQLSourceConfigEntry(override val category: SourceCategory.Value,
```

Comment:
should we refactor `PostgresSQLSourceConfigEntry`  to `PostgreSQLSourceConfigEntry`?